consensus/clique/src/main/java/org/hyperledger/besu/consensus/clique/jsonrpc/methods/Discard.java

```diff
@@ -46,7 +48,13 @@ public String getName() {
   public JsonRpcResponse response(final JsonRpcRequestContext requestContext) {
     checkState(
         validatorProvider.getVoteProviderAtHead().isPresent(), "Clique requires a vote provider");
-    final Address address = requestContext.getRequiredParameter(0, Address.class);
+    final Address address;
+    try {
+      address = requestContext.getRequiredParameter(0, Address.class);
+    } catch (Exception e) { // TODO:replace with JsonRpcParameter.JsonRpcParameterException
+      throw new InvalidJsonRpcParameters(
+          "Invalid address parameter", RpcErrorType.INVALID_ADDRESS_PARAMS, e);
```

Comment:
IIUC, there's two messages related to this error: 
1. The string here, "Invalid address parameter"
2. [The one hardcoded against INVALID_ADDESS_PARAMS (added the previous PR), "Invalid address params"](https://github.com/hyperledger/besu/blob/4e79c810a6c9e94db6168a46e769d82e747da226/ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/response/RpcErrorType.java#L33)

Why do we need both and is there a reason we need all the extra RpcErrorTypes versus dynamically passing the string here?

Reply:
There are times when the message can add some extra context, but for the most part it's redundant.

Reply:
So how does this work, does one override the other or does it respond with both? 
Have you got an example where the message gives extra context?

Reply:
To be honest, I don't really know the full context of how each message is used and where.

Reply:
Updating for completeness...we figured out that this error message is displaying in a DEBUG log stack, whereas the one attached to INVALID_ADDRESS_PARAMS is what the user actually sees. At least in the example tested.